admin-manual/maintenance/cli-import-export.rst

```diff
@@ -513,19 +521,19 @@ The authority record bulk export task has the same options available as the
 archival description export task described :ref:`above <cli-bulk-export-usage>`.
 Some of these options will not be relevant to EAC-CPF exports (e.g. the
 ``--current-level-only`` option, as authority records are not hierarchical; and
-the ``--public`` option, as currently authority records do not have a publication
-status), but otherwise they can be used with this task in the same way as
-described for the archival description export options
+the ``--public`` option, as currently authority records do not have a
+publication status), but otherwise they can be used with this task in the same
+way as described for the archival description export options
 :ref:`above <cli-bulk-export-usage>`. Please refer there for more detailed usage
 notes. Below is an example application, using the ``--criteria`` option:
 
 **Example: using the** ``--criteria`` **option to select only authority records 
 whose entity type is "family"**
 
 First, you will need to know the entity type ID for family. Entity type is a 
-:term:`term` maintained in the Actor entity types :term:`taxonomy` - when elements 
-from a different table in the database are linked to actors, the term ID is used. 
-Here are the term object IDs for the Actor entity types: 
+:term:`term` maintained in the Actor entity types :term:`taxonomy` - when
+    elements from a different table in the database are linked to actors, the
```

Comment:
Hi @currmie it looks like this is where the issue reported by the Travis CI is - my suspicion is that you used an autowrap and it got the wrap wrong and indended these lines? I think if you fix the line indentations in a new commit, it should resolve the errors.

Reply:
Doh! thank you!

> "Excellent! I cried. "Elementary," said he.

Reply:
😆